.github/policies/disallow-edits.yml

```diff
@@ -0,0 +1,41 @@
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+resource: repository
+
+configuration:
+  resourceManagementConfiguration:
+    eventResponderTasks:
+      - description: Close PRs that modify files whose "source of truth" is not in this repo.
+        if:
+          - payloadType: Pull_Request
+          - isAction:
+              action: Opened
+          - or:
+              - filesMatchPattern:
+                  pattern: xml/Microsoft.Extensions*/*.xml
+                  excludedFiles:
+                    - xml/Microsoft.Extensions.FileSystemGlobbing/*.xml
+                    - xml/Microsoft.Extensions.FileSystemGlobbing.Abstractions/*.xml
+                    - xml/Microsoft.Extensions.FileSystemGlobbing.Internal*/*.xml
```

Comment:
What's the reason for these exclusions? I think we should have a comment to explain the exclusion.

Reply:
The reason is this: https://github.com/dotnet/runtime/blob/86995059adb5e76e1701502b980c0bf574451a94/src/libraries/Microsoft.Extensions.FileSystemGlobbing/src/Microsoft.Extensions.FileSystemGlobbing.csproj#L6